<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="Menu_pg">
     <widget class="QLabel" name="Name">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>200</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>23</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sea Raider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_button">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>290</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Начать игру</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Exit_button">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>360</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Connect_pg">
     <widget class="QLabel" name="Name_c">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>159</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>24</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sea Raider</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Ip_line">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>250</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Port_line">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>320</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>55555</string>
      </property>
     </widget>
     <widget class="QLabel" name="IpLab">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>220</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ip адрес</string>
      </property>
     </widget>
     <widget class="QLabel" name="PortLab">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>290</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Порт</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connect_button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>370</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Присоединиться</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RegLog_pg">
     <widget class="QLabel" name="Name_rl">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>210</y>
        <width>200</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>23</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sea Raider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Reg_button">
      <property name="geometry">
       <rect>
        <x>286</x>
        <y>290</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Зарегистрироваться</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Log_button">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>350</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Lobby_pg">
     <widget class="QLabel" name="PlayersLb">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Игроки в лобби</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Quit_button">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>499</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Отсоединиться и выйти</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>431</width>
        <height>441</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="WidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>429</width>
         <height>439</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="YourLb">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>90</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <italic>false</italic>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Ваша статистика</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="UserLb">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>130</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="WinLb">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>220</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Побед:</string>
      </property>
     </widget>
     <widget class="QLabel" name="WinLb2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>220</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="LoseLb">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>274</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Поражений:</string>
      </property>
     </widget>
     <widget class="QLabel" name="LoseLb2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>270</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="MatchLb">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>320</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Всего матчей:</string>
      </property>
     </widget>
     <widget class="QLabel" name="MatchLb_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>320</y>
        <width>49</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Select_pg"/>
    <widget class="QWidget" name="Game_pg"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>Exit_button</tabstop>
  <tabstop>Ip_line</tabstop>
  <tabstop>Port_line</tabstop>
  <tabstop>Reg_button</tabstop>
  <tabstop>Log_button</tabstop>
  <tabstop>Quit_button</tabstop>
  <tabstop>start_button</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
